File: app2/src/main/java/ru/neverlands/anclient/model/AncCell.java
package ru.neverlands.anclient.model;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class AncCell {
    @PrimaryKey
    @NonNull
    public String RegNum;
    public String Label;
    public int Cost;
    public long Visited;
    public long Verified;
}

File: ANClient/ExtMap/AncCell.cs
using System;

namespace ANClient.ExtMap
{
    public class AncCell
    {
        public string RegNum { set; get; }
        public string Label { set; get; }
        public int Cost { set; get; }
        public DateTime Visited { set; get; }
        public DateTime Verified { set; get; }
    }
}
